libs/state/src/cart/effects/cart.effects.spec.ts

```diff
@@ -25,8 +25,8 @@ describe('Daffodil | State | Cart | CartEffects', () => {
   
   let mockCart: Cart;
 
-  let cartFactory: CartFactory;
-  let productFactory: ProductFactory;
+  let cartFactory: DaffCartFactory;
+  let productFactory: DaffProductFactory;
```

Comment:
This isn't needed

Reply:
resolved: https://github.com/graycoreio/daffodil/pull/115/commits/d0e1e198882fa8815bc4e3b1a70a13c2be44d3db